hardhat.config.ts

```diff
@@ -8,6 +8,7 @@ import "@openzeppelin/hardhat-upgrades"
 import "@typechain/hardhat"
 import * as dotenv from "dotenv"
 import "hardhat-contract-sizer"
+import "hardhat-change-network"
```

Comment:
is this a preparation for going 'multichain'?

Reply:
Could be but not main intention of this PR.